cli/command/volume/testdata/volume-prune-no.golden

```diff
@@ -1,2 +1,2 @@
 WARNING! This will remove anonymous local volumes not used by at least one container.
-Are you sure you want to continue? [y/N] 
+Are you sure you want to continue? [y/N] 
```

Comment:
This is strange...testing it manually it does end on a new line, but maybe the `gotest.tools/v3/golden` package is stripping some of the characters for some reason.

```shell
⋊> ~/G/docker-cli on prompt-termination ◦ ./build/docker system prune           12:56:53
WARNING! This will remove:
  - all stopped containers
  - all networks not used by at least one container
  - all dangling images
  - unused build cache

Are you sure you want to continue? [y/N] ^C
⋊> ~/G/docker-cli on prompt-termination ◦ ./build/docker system prune           12:56:59
WARNING! This will remove:
  - all stopped containers
  - all networks not used by at least one container
  - all dangling images
  - unused build cache

Are you sure you want to continue? [y/N] n
```

Reply:
Just to confirm, removing the new line print in the prompt function ends up with this:
```shell
> ~/G/docker-cli on prompt-termination ⨯ bash                                             13:14:15
[benehiko@benehiko docker-cli]$ ./build/docker system prune
WARNING! This will remove:
  - all stopped containers
  - all networks not used by at least one container
  - all dangling images
  - unused build cache

Are you sure you want to continue? [y/N] ^C[benehiko@benehiko docker-cli]$
```

Reply:
```diff
diff --git a/cli/command/utils.go b/cli/command/utils.go
index ae93dbc30a..52cb98bd4a 100644
--- a/cli/command/utils.go
+++ b/cli/command/utils.go
@@ -123,7 +123,7 @@ func PromptForConfirmation(ctx context.Context, ins io.Reader, outs io.Writer, m
        select {
        case <-notifyCtx.Done():
                // print a newline on termination
-               fmt.Fprintln(outs, "")
+               // fmt.Fprintln(outs, "")
                return false, ErrPromptTerminated
        case r := <-result:
                return r, nil
diff --git a/cli/command/volume/testdata/volume-prune-terminate.golden b/cli/command/volume/testdata/volume-prune-terminate.golden
index 8e918f9c9e..9f7c8c76a5 100644
--- a/cli/command/volume/testdata/volume-prune-terminate.golden
+++ b/cli/command/volume/testdata/volume-prune-terminate.golden
@@ -1,2 +1,2 @@
 WARNING! This will remove anonymous local volumes not used by at least one container.
-Are you sure you want to continue? [y/N]
+Are you sure you want to continue? [y/N]
\ No newline at end of file
```

Reply:
Oh my, you're going to laugh. This is correct :)
https://github.com/docker/cli/blob/9bc91a81236b182e0dadb91e0d5d8d223092ab9e/cli/command/volume/prune_test.go#L168-L178

We don't add carriage returns to test inputs :see_no_evil: 

The reason we only picked this up now is because in the old implementation of the `volume prune` command we used to always print a new line inside the cobra command run function for this command. 

https://github.com/docker/cli/blob/master/cli/command/volume/prune.go#L36

Since `runPrune` would return an error on prompt error or `n`.
https://github.com/docker/cli/blob/master/cli/command/volume/prune.go#L85

Reply:
So, should we change the test here to add a newline?

Reply:
We don't do it in any of the other tests for the prompt input. I'd prefer not to change it.

Reply:
But now, does anything happen if there is just an `n` without a newline? `PromptForConfirmation` doesn't return if the user only writes an `n`, right?

Reply:
In the test it creates a reader with the value already. So when prompt is called, the value is immediately read. The behavior here is not the same as `stdin`.

Reply:
I see, thanks!